Userland/Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -919,6 +919,24 @@ double to_integer_or_infinity(double number)
     return integer;
 }
 
+// https://tc39.es/proposal-temporal/#sec-tointegerifintegral
```

Comment:
This doesn't belong here as long as it's part of the Temporal proposal, see `Temporal/AbstractOperations.cpp/h`